app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+end
```

Comment:
- [ ] You did well  setting the controllers however it is required to have an index and a show actions. Kindly add the index action.